source/session.ts

```diff
@@ -624,7 +628,7 @@ export class Session {
         throw this.getErrorFromResponse(response);
       }
       try {
-        return this.decode(response, {}, decodeDatesAsIso);
+        return this.decode(response, {}, this.decodeDatesAsIso || decodeDatesAsIso);
```

Comment:
This doesn't allow you to override a `true` default for decodeDatesAsIso.

Reply:
Ahhhh I see! Will correct.

Reply:
There we go!